### DISCUSSION TITLE
Filtering for Blocked Users

### DISCUSSION THREAD

**Discussion by bauclair** [jdhitsolutions/PSBluesky#26] [Ideas]
Is there/will there be a way to filter for users that are blocked?  For example, I'd like to query for a list of users that have not already been blocked (whether individually or via moderation list) by my Bluesky account.  Is this doable?  Please advise.

**Comment by jdhitsolutions** [comment]
It looks like you can query for a list of [users you already block](https://docs.bsky.app/docs/api/app-bsky-graph-get-blocks). I also see an API to [show lists you are blocking](https://docs.bsky.app/docs/api/app-bsky-graph-get-list-blocks). Are these what you have in mind?

**Reply by bauclair** [reply]
Both of these look useful.  Can they be added as PSBluesky module commands?  It would make my filtering tasks a lot more effective, and would provide some unique capabilities that no other Bluesky client or tool I've found currently has.

**Reply by jdhitsolutions** [reply]
I can put this on the roadmap. Can't guarantee a timeframe.

**Reply by bauclair** [reply]
Thank you so much!

**Comment by jdhitsolutions** [comment]
Added a few commands which should be a part of the 2.2.0 release.
